lmfdb/hilbert_modular_forms/q_expansion.py

```diff
@@ -41,7 +41,7 @@ def qexpansion(field_label=None):
             v = next(S)
             continue
 
-        if field_label is None or not field_label != v["field_label"]:
+        if field_label is None or field_label == v['field_label']:
```

Comment:
this looks dubious

Reply:
I agree.  I'll take a look.  Thanks for the PR.

Reply:
The file has a comment at the top that "As of August 2017 only one newform had this field, so apart from testing this has not been run."  But reading the code it's clearly a bug, so I'll fix it in the PR.